### DISCUSSION TITLE
I have access to repos in multiple organizations as well as my own. It would be nice to be able to tap Repos under My Work and then filter by organization as well as repo type. It'd help me and others get to target repos more quicky.

### DISCUSSION THREAD

**Discussion by smittytone** [community/community#53663] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.109.0 (96776079)
Device: iPad Pro 11-inch 3 (iOS 16.4.1)
</details>

**Comment by taki-on** [comment]
Thank you for the feature request! We also think filtering repositories under My Work will be very helpful and we currently track the feature in our backlog. I hope we can deliver the feature soon. Please stay tuned!

**Reply by jsoref** [reply]
Please mark this as the answer
